R/brood_check.R

```diff
@@ -1250,15 +1263,14 @@ check_clutch_type_order <- function(Brood_data, approved_list){
               ErrorOutput = unlist(error_output)))
 
   # Satisfy RCMD checks
-  Brood_err <- NULL
   approved_list <- NULL
 
 }
 
 
 #' Compare species of parents
 #'
-#' Check that the parents of broods are of the same species. Broods with parents of different species are flagged as 'warning' because multi-species broods are known to exist.
+#' Check that the parents of broods are of the same species. Common, biologically possible multi-species broods are flagged as ‘warning’. Other combinations of species are flagged as ‘potential error’.
```

Comment:
Is 'multi-species broods' the right term to use here? For me a 'multi-species brood' would be one with chicks from two different species (e.g. when a GT raises some BT eggs in a clutch it took over). I think here we're referring to *hybrid* broods right?

```suggestion
#' Check that the parents of broods are of the same species. Common, biologically possible hybrid broods are flagged as ‘warning’ (e.g. FICHYP and FICALB). Other combinations of species are flagged as ‘potential error’.
```